aptos-move/framework/aptos-stdlib/sources/event.spec.move

```diff
@@ -0,0 +1,18 @@
+spec aptos_std::event {
+    // ****************** SPECIFICATIONS *******************
+    spec module {} // switch documentation context to module
+
```

Comment:
```suggestion
```

Reply:
We may still need it so that the document generator can properly work later.

Reply:
This seems a bit weird as other spec files do not seem to have this context switch comment. So my guess is that the docgen does not rely on it anymore, but I might be wrong.